### PR TITLE
newdevice: Add extra Sophos discovery

### PR DESCRIPTION
**Pull request by Rosiak**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [X] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/)
- [X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

New firmware..

### CODE REVIEW

File: tests/snmpsim/sophos4.snmprec [hunk 1]
```diff
@@ -0,0 +1 @@
+1.3.6.1.2.1.1.1.0|4|Linux g4dc0e52
```
Comment (murrant) [inline tests/snmpsim/sophos4.snmprec]: Is the actually sysDescr from the device?

Comment (Rosiak) [inline tests/snmpsim/sophos4.snmprec]: Nope, but don't have other than that right now :(

Comment (laf) [inline tests/snmpsim/sophos4.snmprec]: We could look at using `&& preg_match('/Linux ([\S]+) ([0-9\-.]{1,11})(g[\w\d]{7})(.*)/', $sysDescr)`

That matches the test data I found from an old issue: https://gist.github.com/laf/67cee069eb3db4af26dccdb67a7da3f5 but we'd need to find others potentially.

Comment (murrant) [inline tests/snmpsim/sophos4.snmprec]: Thanks for the regex @laf  I was thinking something like this but gave me a good head start :)

Comment (laf) [inline tests/snmpsim/sophos4.snmprec]: Np :)

**Pull request closed without merging**